index.html

```diff
@@ -107,15 +116,22 @@
             </a>
           </div>
           <div class="col-md-6 col-sm-12 text-center align-items-center" style="margin-top: 11px">
-            <h1 class="heading">Sunbird Translation</h1>
-            <h2 class="subheading">Translate Ugandan languages</h2>
+            <h1 class="heading">SunbirdAI Translate</h1>
           </div>
-        </div>
+        </div> 
+      
+      
       </header>
-    </section>
+    </section> 
 
+    
+    
+    &nbsp;
     <section>
-      <div class="container-xl px-5 mt-5 py-1">
+
+      <!-- <div class="container-xl px-5 mt-5 py-1"> -->
+        <div class="mdc-card mdc-card--outlined">
```

Comment:
I'm not sure about the look of this `mdc-card--outlined` card surrounding the translation section......